cpp/src/copying/slice.cpp

```diff
@@ -1,7 +1,5 @@
 /*
- * Copyright 2019 BlazingDB, Inc.
- *     Copyright 2019 Christian Noboa Mardini <christian@blazingdb.com>
- *     Copyright 2019 William Scott Malpica <william@blazingdb.com>
+ * Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
Put this below the BlazingDB copyrights, don't delete theirs.

Reply:
It's an entirely new implementation and file. Not a single line of source was preserved, as such, I don't see a reason to keep the Blazing license.